src/utils.ts

```diff
@@ -35,3 +37,28 @@ export const DEFAULT_PROMPT: string =
   `Based on the URL: "{{tabURL}}" and title: "{{tabTitle}}", ` +
   `classify the browser tab type as one of the following: "{{types}}". ` +
   `Respond with only the classification keyword from the list.`;
+
+export function matchesRule(url: URL, rule: FilterRuleItem) {
+  const host = url.host;
+  const { type, rule: value } = rule;
+  if (!value) {
+    return false;
```

Comment:
maybe it's better move to the top of this function instead of get host first

Reply:
thanks! done😁